### DISCUSSION TITLE
feat: Adding AWS Cognito Authentication

### DISCUSSION THREAD

**Discussion by Shekswess** [langfuse/langfuse#2101] [Ideas]
### Describe the feature or potential improvement

The idea is to add AWS Cognito Authentication that could be easily used for authentication of self-hosted Langfuse server on AWS.

**Comment by marcklingen** [comment]
thanks, opened an issue here: https://github.com/langfuse/langfuse/issues/2107

**Comment by marcklingen** [comment]
Added in v2.45.1
